quinn-udp/Cargo.toml

```diff
@@ -14,14 +14,17 @@ workspace = ".."
 all-features = true
 
 [features]
-default = ["log"]
-# Write logs via the `log` crate when no `tracing` subscriber exists
+default = ["tracing", "log"]
+# Write logs via the `log` crate trough the `tracing` crate when no `tracing`
```

Comment:
Maybe shorten this a bit to "Configure tracing to log events via `log` if no `tracing` subscriber exists"?

Reply:
Works for me. I updated the comment across all `Cargo.toml` files to stay consistent. Let me know what you think @djc.